pages/wg/outreach.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: page
+title: Outreach Working Group
+subtitle:
+permalink: /wg/outreach/
+---
+
+# Overview
+
+The US-RSE Outreach Working Group is dedicated to broadening the US-RSE's
+engagement with the research software engineer (RSE) and wider computational
+science and engineering (CSE) communities. We strive to raise awareness of
+US-RSE as an association and RSE as a professional designation through
+engagement with industry, academia, national labs, and students.
+
+Those interested in contributing to this goal are encouraged to join our
+bi-weekly virtual meetings. To get involved, visit the
+[`#outreach`](https://usrse.slack.com/messages/outreach) channel on the US-RSE
+slack or contact the
+<a href="mailto:outreach_wg@us-rse.org">Outreach working group list</a>.
+
+# Work Products
+
+The US-RSE Outreach working group is new as of 2022. The following events were
+either completed before the creation of the official working group or are
+initiatives for future work.
+
+## Existing Efforts
+
+### RSE Stories
+
+Created by Vanessa Sochat, the [RSE Stories podcast](https://us-rse.org/rse-stories/)
+aims to cast a spotlight on the differing backgrounds and paths professionals
+have taken to become RSEs and help shape our understanding of what an RSE really is.
+
+The podcast has existed since September 2019 and is looking for more volunteers
+both for interviewees and hosts.
+
+### Conference and Workshop Participation
+
+US-RSE has had a presence at several conferences and workshops throughout
+its lifetime. For example:
+
+- Supercomputing 2021: [RSE-in-HPC Workshop](https://us-rse.org/rse-hpc-2021/), Student Program Orientation
+- PEARC21: [RSE Career Panel](https://us-rse.org/2021-07-20-pearc21-panel/)
+- Supercomputing 2020: [RSE-in-HPC Workshop](https://us-rse.org/rse-hpc-2020/)
+- PEARC20: [Research Software Engineers Community Workshop](https://us-rse.org/events/2020/2020-07-pearc20/)
+- PEARC19: Building a Community of Research Software Engineers Birds of a Feather - [Proposal](https://ssl.linklings.net/conferences/pearc/pearc19_program/views/includes/files/bof111s1.pdf), [Event](https://us-rse.org/events/2019/pearc-2019/)
+- EDUCAUSE 2019: [Supporting Research Software and Research Software Engineers Poster](https://events.educause.edu/annual-conference/2019/agenda/supporting-research-software-development)
+
+### External Collaboration
+
+US-RSE has collaborated with other organizations in the past for events
+such as the [Early Career Panel](https://academicdatascience.org/cdn/early-career-panel)
+([Academic Data Science Alliance](https://academicdatascience.org/)).
+
+## Future Directions
+
+### Conference and Workshop Participation
+
+US-RSE will continue to engage in conferences and workshops. We aim to
+broaden our presence at previously unrepresented conferences, including
+domain-specific conferences and general technical conferences.
+
+Towards this goal, the Outreach working group is putting together a call for
```

Comment:
Can someone (@mrmundt?) set a reminder to update this once the deadline is over? These things make me nervous of rotting on the page?

Reply:
Yup! I'll put a reminder on my calendar!